### PR TITLE
docs: remove margin with list

### PR DESCRIPTION
**Pull request by stefanoScalzo**
## Related Issue
Closes SAP/fundamental-styles#915

## Description
This pr will remove the extra padding which is no longer necessary in tablet mode. Design approved by @estelaxu3. 

### Before:
Screen Shot 2020-04-28 at 7 08 30 PM

### After:
Screen Shot 2020-04-28 at 7 08 26 PM

### CODE REVIEW

**Comment by rengare** [comment]
Popover with Header and fd-menu looks odd to me @InnaAtanasova @droshev what do you think?

**Comment by rengare** [comment]
@stefanoScalzo you can close this PR because @salarenko made new one (#894) which fix the issue

**Comment by stefanoScalzo** [comment]
> @stefanoScalzo you can close this PR because @salarenko made new one (#894) which fix the issue

This pr is to make sure all ul dont get affected which is still needed for future in case someone uses a ul without a modifier class

File: docs/_sass/_docs-main-content.scss [hunk 1]
```diff
@@ -22,7 +22,7 @@
         margin: 10px 0 10px 0;
     }
 
-    ul{
+    > ul {
         @media screen and (max-width: 950px) {
             padding-left: 0;
             margin-left: 3rem;
```
Comment (InnaAtanasova) [inline docs/_sass/_docs-main-content.scss:28]: I don't think we should style ul in the documentation. In my opinion this should be removed and if the documentation page needs styling for ul it should rely on a specific class

Comment (stefanoScalzo) [reply docs/_sass/_docs-main-content.scss]: fixed

**Comment by InnaAtanasova** [comment]
Screen Shot 2020-05-01 at 1 56 16 PM
Screen Shot 2020-05-01 at 1 56 05 PM
Screen Shot 2020-05-01 at 1 55 09 PM

**Comment by stefanoScalzo** [comment]
> If this is not in the scope of this PR I think a new issue should be created. Then you put the list inside a popover the borders overlap resulting in a darker border color.
> 
> Screen Shot 2020-05-01 at 1 56 16 PM
> 
> Screen Shot 2020-05-01 at 1 56 05 PM
> 
> Screen Shot 2020-05-01 at 1 55 09 PM

I've also noticed this issue and it's not only for this, we have overlapping issues in many places i will create an issue

File: docs/pages/components/notification.md [hunk 2]
```diff
@@ -8,8 +8,7 @@ permalink: components/notification.html
 folder: components
 summary:
 ---
-
-{: .docs-intro}
+{: .docs-intro .docs-intro--def-margin}
```
Comment (droshev) [inline docs/pages/components/notification.md:11]: this should not be part of this PR

Comment (stefanoScalzo) [reply docs/pages/components/notification.md:11]: The PR changes all the margin-left's in the documentation. There is nothing touching the library so it should be all part of same pr as it is all in the same scope
@droshev
